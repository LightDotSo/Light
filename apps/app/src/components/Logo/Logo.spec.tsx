import { expect } from "@jest/globals";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

import { Logo } from "./Logo";

it("Renders Logo", () => {
  const { asFragment } = render(<Logo />);
  expect(asFragment()).toMatchSnapshot();
});
