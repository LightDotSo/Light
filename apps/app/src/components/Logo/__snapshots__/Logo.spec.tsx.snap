// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Logo 1`] = `
<DocumentFragment>
  <svg
    class="fill-contrast-high"
    fill="none"
    height="205"
    viewBox="0 0 628 205"
    width="628"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0_215_1308)"
    >
      <path
        d="M294.444 167.693V37.3091H308.466V167.693H294.444Z"
      />
      <path
        d="M329.682 58.7136V37.3065H343.508V58.7136H329.682ZM329.682 167.694V70.3931H343.508V167.694H329.682Z"
      />
      <path
        d="M359.864 114.372C359.864 87.1268 374.859 69.613 401.73 69.613C418.675 69.613 431.524 77.9798 435.812 90.2399H436.589V71.5578H449.05V163.608C449.05 189.685 432.305 202.721 406.988 202.721C382.455 202.721 365.121 189.877 365.121 167.694H378.951C378.951 183.455 386.938 190.269 407.183 190.269C427.828 190.269 435.031 182.874 435.031 163.608V139.088H434.835C430.359 150.18 418.286 159.131 401.538 159.131C375.055 159.142 359.864 141.617 359.864 114.372ZM404.849 147.067C422.179 147.067 435.031 137.143 435.031 117.289V111.455C435.031 91.6042 421.982 81.6772 404.068 81.6772C385.766 81.6772 373.886 89.0752 373.886 114.372C373.886 139.864 386.154 147.067 404.849 147.067Z"
      />
      <path
        d="M470.853 167.694V37.3101H484.875V89.478H485.652C489.352 80.331 499.282 69.6275 518.757 69.6275C543.879 69.6275 554.589 86.3649 554.589 108.356V167.708H540.756V111.843C540.756 91.02 533.745 81.873 514.27 81.873C494.994 81.873 484.861 90.8241 484.861 112.427V167.694H470.853Z"
      />
      <path
        d="M583.601 140.449V83.6256H567.44V71.5579H583.778L587.09 47.0413H597.619V71.5579H628.001V83.6256H597.619V141.229C597.619 152.513 602.88 155.045 615.341 155.045H628.001V166.333C622.603 167.327 617.129 167.848 611.641 167.89C593.919 167.89 583.601 160.883 583.601 140.449Z"
      />
      <path
        d="M48.4033 15.5257C51.5365 13.5458 54.7751 11.7374 58.1046 10.1086C61.0964 8.65727 64.1631 7.36191 67.3049 6.22262C78.6042 2.0938 90.5431 -0.012589 102.574 7.47092e-05C121.758 -0.0231261 140.561 5.35786 156.825 15.5257H48.4033Z"
      />
      <path
        d="M94.0552 31.4903H176.495C178.872 33.9624 181.123 36.5494 183.248 39.2513H21.8903C26.3667 33.5604 31.432 28.3575 37.0016 23.7294H102.29C99.4043 26.1632 96.6553 28.7541 94.0552 31.4903V31.4903Z"
      />
      <path
        d="M197.224 62.9914H7.94971C10.2053 57.5956 12.9189 52.4026 16.0608 47.4694H81.349C79.7478 49.9802 78.2882 52.5817 76.8976 55.2304H193.564C194.886 57.7448 196.117 60.3282 197.224 62.9914Z"
      />
      <path
        d="M203.927 86.7023H1.26556C2.06872 81.4416 3.28268 76.2518 4.89631 71.1804H70.1847C69.3678 73.7203 68.6888 76.3218 68.0679 78.9414H202.406C202.99 81.4958 203.52 84.079 203.927 86.7023Z"
      />
      <path
        d="M65.2912 102.5C65.2912 102.558 65.2912 102.612 65.2912 102.671H205.133C205.133 105.283 204.999 107.866 204.799 110.432H0.333468C0.133777 107.812 -0.000488281 105.174 -0.000488281 102.5C-0.000488281 99.9602 0.11927 97.4205 0.304438 94.9097H65.589C65.4111 97.4168 65.2912 99.9457 65.2912 102.5Z"
      />
      <path
        d="M68.1768 126.396H202.308C201.678 129.023 200.952 131.61 200.129 134.157H4.98698C3.35971 129.087 2.13244 123.897 1.31635 118.635H66.6045C67.0184 121.262 67.5704 123.842 68.1768 126.396Z"
      />
      <path
        d="M188.864 157.883H16.2908C13.1202 152.953 10.3799 147.76 8.09985 142.361H73.388C74.5099 145.017 75.7808 147.589 77.1169 150.122H193.399C191.997 152.782 190.486 155.369 188.864 157.883Z"
      />
      <path
        d="M87.4389 166.09C89.612 168.794 91.9231 171.384 94.3629 173.851H176.182C173.532 176.589 170.733 179.18 167.798 181.612H37.4217C31.7964 176.992 26.6775 171.789 22.1508 166.09H87.4389Z"
      />
      <path
        d="M114.234 189.815C117.199 191.663 120.257 193.358 123.394 194.895C125.402 195.86 127.461 196.738 129.538 197.576H140.902C128.722 202.489 115.71 205.009 102.576 205C90.5449 205.014 78.6059 202.909 67.3065 198.781C64.1598 197.63 61.0931 196.334 58.1062 194.895C54.9685 193.358 51.9111 191.663 48.9457 189.815H114.234Z"
      />
    </g>
    <defs>
      <clippath
        id="clip0_215_1308"
      >
        <rect
          fill="white"
          height="205"
          width="628"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;
